src/test/app/utils/momentFormatter.ts

```diff
@@ -0,0 +1,31 @@
+import { expect } from 'chai'
+import * as moment from 'moment'
+
+import { MomentFormatter } from 'app/utils/momentFormatter'
+
+describe('MomentFormatter', () => {
+
+  describe('formatDate', () => {
+    it('format moment value to date string', () => {
+      expect(MomentFormatter.formatDate(moment('2017-01-01'))).to.eq('2017-01-01')
+    })
+  })
+
+  describe('formatLongDate', () => {
+    it('format moment value to date string', () => {
+      expect(MomentFormatter.formatLongDate(moment('2017-01-01'))).to.eq('1 Jan 2017')
+    })
+  })
+
+  describe('formatLongDateWithLongMonth', () => {
+    it('format moment value to date and time string', () => {
+      expect(MomentFormatter.formatLongDateWithLongMonth(moment('2017-01-01'))).to.eq('1 January 2017')
+    })
+  })
+
+  describe('formatLongDateAndTime', () => {
+    it('format moment value to date and time string', () => {
+      expect(MomentFormatter.formatLongDateAndTime(moment('2017-01-01T11:22:33'))).to.eq('1 Jan 2017 at 11:22am')
```

Comment:
You probably want to make this change too:
https://github.com/hmcts/cmc-citizen-frontend/pull/40/files#diff-3f2c7e6e59cd8f5f1fdb8c1d707b75bfR4

Reply:
This PR is just to add missing tests to existing behaviour. We will change that if required and unit tests will reflect then.